run_base_images_build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PYTHON_VERSIONS=("3.11" "3.12" "3.13")
+
+for v in ${PYTHON_VERSIONS[@]};
+do
+  echo "Creating base image for Python $v"
+  podman build --build-arg version=$v -f Dockerfile.base -t terratorch_python_$v
```

Comment:
Do we have any dependency on podman? I would rather use docker as in the end there's mappings for the docker cli to the podman backend.